kimhaejung/app.js

```diff
@@ -35,6 +36,89 @@ app.get("/ping", function (req, res, next) {
   res.json({ message: "pong" });
 });
 
+app.post("/users/signup", async (req, res, next) => {
+  const { name, email, profileImage, password } = req.body;
+
+  await appDataSource.query(
+    `INSERT INTO users(
+      name,
+      email,
+      profile_image,
+      password
+    ) VALUES (?, ?, ?, ?);
+    `,
+    [name, email, profileImage, password]
+  );
+  res.status(201).json({ message: "userCreated" });
+});
+
+app.post("/posts", async (req, res, next) => {
+  const { title, content, userId, imageUrl } = req.body;
+
+  await appDataSource.query(
+    `INSERT INTO posts(
+      title,
+      content,
+      user_id,
+      image_url
+    ) VALUES (?, ?, ?, ?);
+    `,
+    [title, content, userId, imageUrl]
+  );
+  res.status(201).json({ message: "postCreated" });
+});
+
+app.get("/posts/look", async (req, res, next) => {
```

Comment:
엔드포인트를 작성할 때는 look과 같은 동사형 엔드포인트보다는 특정 리소스에 대해 명사형으로 작성해주는 것이 좋습니다.

HTTP 메서드 자체가 이미 어떤 행위를 가리키는 동사형으로 이루어져있기 때문입니다.

따라서 게시글을 조회하는 엔드포인트를 구현할 때는 다음과 같이 작성할 수 있습니다.

```suggestion
// 전체 게시글 조회
app.get("/posts", async (req, res, next) => {
// 전체 게시글 조회
app.get("/posts", async (req, res, next) => {

// 단일 게시글 조회
app.get("/post/:postId", async (req, res, next) => {
```